TriviaNation.Student/App.xaml.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+using TriviaNation.Core.Drivers;
+using TriviaNation.Core.Models;
+using TriviaNation.Core.Util.CustomExceptions;
+
+namespace TriviaNation
+{
+	/// <summary>
+	/// Interaction logic for App.xaml
+	/// </summary>
+	public partial class App : Application
+	{
+		public App() : base()
+		{
+		}
+
+
+		/// <summary>
+		/// Attempts to find the user in the database.
+		/// </summary>
+		/// <param name="userId"></param>
+		/// <param name="password"></param>
+		/// <returns>Null on succes, or an error message if not successful</returns>
+		public static async Task OnLogin(string userId, string password)
+		{
+			LoginMessage = null;
+
+			using (var db = new LoginDriver())
+			{
+				try
+				{
+					var user = await db.Login(userId, password, "Student");
+					
+					if ((user as StudentUser) == null)
+					{
+						throw new Exception();
+					}
+
+                    if(user.Name == "" || user.Email == "")
+                    {
+                        throw new Exception();
+                    }
+
+					Application.Current.Properties.Add("LoggedInUserId", user.Email);
+					Application.Current.Properties.Add("LoggedInUserName", user.Name);
+				}
+				catch (Exception ex)
+				{
+					LoginMessage = "Unable to Login.";
+				}
+			}
+
+		}
+
+		public static string LoginMessage { get; set; }
+
+		public static void OnLogout()
+		{
+			Application.Current.Properties.Remove("LoggedInUserId");
+			Application.Current.Properties.Remove("LoggedInUserName");
+
+			Application.Current.Shutdown();
```

Comment:
Do we want the entire application to close on logout, or do we just want to return to the login screen?

Reply:
I'm fine with this closing on logout for now. We can implement returning to the log in screen later.

Reply:
Sounds good to me.